### PR TITLE
Update formatting

### PR DESCRIPTION
**Pull request by Microwave-6443**
Prettier didn't like the previous formatting.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
-    "validator": "^13.9.0"
-  }
-}
\ No newline at end of file
+	"name": "bssr-nodes-bot",
+	"version": "1.0.0",
+	"description": "Main bot for BSSR Nodes.",
+	"main": "index.js",
+	"scripts": {
+		"start": "npm i && node ."
```
Comment (ninsaccplayzyt) [inline package.json:7]: We don't use this to start the bot and that so not really needed

**Review by ninsaccplayzyt — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Microwave-6443) [reply package.json:7]: It was already there, so I didn't really bother to remove it.

Comment (ninsaccplayzyt) [reply package.json:7]: Probably from dbh bot, never noticed it, never used it

Comment (Microwave-6443) [reply package.json:7]: Alright.

**Review by ninsaccplayzyt — APPROVED** [review]
LGTM!

**Pull request merged**